README.md

```diff
@@ -359,7 +359,7 @@ Enable access to a service or service plan for one or all orgs
 
 * `org`: *Optional.* The organization to target (required if not set in the source config)
 * `space`: *Optional.* The space to target (required if not set in the source config)
-* `service`: *Required.* The marketplace service name to enable
+* `service_broker`: *Required.* The marketplace service name to enable
```

Comment:
This description should probably change too. Maybe "The name of the service broker to enable"? Same with the docs for disable.